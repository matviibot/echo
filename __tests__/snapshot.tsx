import * as React from 'react';
import { render } from '@testing-library/react';

import '@testing-library/jest-dom';

function Page(): React.JSX.Element {
  return <h1>Redefining SaaS</h1>;
}

describe('Home', () => {
  it('renders homepage unchanged', () => {
    const { container } = render(<Page />);

    expect(container).toMatchSnapshot();
  });
});
