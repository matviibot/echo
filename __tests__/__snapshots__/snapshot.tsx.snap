// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home renders homepage unchanged 1`] = `
<div>
  <h1>
    Redefining SaaS
  </h1>
</div>
`;
